docs/coordinates/matchsep.rst

```diff
@@ -81,23 +81,17 @@ telescope operator to move from a bright star to a fainter target.)::
 To extend the concept of spherical offsets, `~astropy.coordinates` has
 machinery to create distinct frames that are centered on a specific point.
 These are known as "astrometric frames" (as they are a convenient way to create
-a locally "flat" frame on relatively small fields suitable for astrometry). One
-complication of astrometric frames is that they cannot be created in the same
-manner as other frames, because astrometric frames are generated on-the-fly
-given a particular "origin" frame. The 
-`~astropy.coordinates.make_astrometric_cls` function instead must be used to
-generate the class that creates such a frame::
-
-    >>> from astropy.coordinates import make_astrometric_cls, ICRS
-    >>> AstrometricICRS = make_astrometric_cls(ICRS)
+a locally "flat" frame on relatively small fields suitable for astrometry)::
```

Comment:
This was already introduced in the earlier PR by @eteq ... I'm not sure it's appropriate to comment on this here, given that this PR might be controversial and not get merged. Anyways:

I find it confusing to call the astrometric frames as defined here locally flat.
To me this suggests that they are the local TAN projection.
But they are not, they are spherical coordinate systems.

So I suggest to change the basic description to "frames rotated to put the point lon=0, lat=0 on a given center" or something, but avoid calling them flat.